src/ctapipe_io_lst/evb_preprocessing.py

```diff
@@ -1,25 +1,45 @@
-from enum import IntEnum
+from enum import IntEnum, IntFlag
 from .constants import TriggerBits
 from collections import defaultdict
 
 class EVBPreprocessing(IntEnum):
     """
     The preprocessing steps that can be applied by EVB.
 
-    The values of this Enum is the index of this step in the tdp_action array.
+    The values of this Enum is the index of this step in the ttype_pattern array.
 
-    See EVB ICD section in
+    Notes
+    -----
+    This was supposed to be documented in the EVB ICD:
     https://edms.cern.ch/ui/file/2411710/2.6/LSTMST-ICD-20191206.pdf
+    But that document doest match the current EVB code.
+
+    As of 2024-01-26, there is a bug in EVB that the ttype_pattern and
+    tdp_action arrays are actually mixed up in the camera_configuration
+    object.
```

Comment:
Can this be explained a bit more? The meaning of both items, and the nature (and consequences) of the problem.

Reply:
EVB just filled the real ttype_pattern into tdp_action and vice-versa. So the consequence is that if you want to know one, you actually have to look into the other field.

Reply:
Is that actually fixed now in the code anywhere? Or simply not yet used?

Reply:
I checked again and realized I had not updated the `get_evb_preprocessings` function below. This is now done and I added a test.

As far as I know, this bug is not yet fixed in EVB.